RunJavaUtils.py

```diff
@@ -128,23 +129,21 @@ def Run_Java_File(destName, package, baseName):
         fullClassName = os.path.join(package, className)
         invokeName = package + "." + invokeName            
     try:
-        subprocess.check_output(["javac", destName], stderr=True)
-    except subprocess.CalledProcessError:
-        output = ["build error"]
+        subprocess.check_output(["javac", destName])
+    except subprocess.CalledProcessError as e:
+        output = ["build error: " + str(e.output)]
```

Comment:
For this kind of thing, I didn't think `e.output` was necessary. I usually just write `str(e)`.